### PR TITLE
Add silent kick and silent ban commands

### PR DESCRIPTION
**Pull request by ammarfaizi2**
Sometimes it's annoying to see a lot of ban and kick commands being
invoked to get rid of spammers. As such this issue, we introduce
silent version of these commands.

The silent version will perform the action silently and will delete
the command message sent by the invoker, including replied message.

Suggested-by: Arthur Lapz <arthurlapz@gmail.com>
Signed-off-by: Ammar Faizi <ammarfaizi2@gmail.com>

### CODE REVIEW

File: src/gwbot/modules/003_admin/admin.c [hunk 1]
```diff
@@ -36,18 +36,20 @@ int GWMOD_SHUTDOWN_DEFINE(003_admin, struct gwbot_state *state)
...
-	ADM_CMD_WARN	= (1u << 3u),
-	ADM_CMD_MUTE	= (1u << 4u),
-	ADM_CMD_TMUTE	= (1u << 5u),
-	ADM_CMD_UNMUTE	= (1u << 6u),
-	ADM_CMD_PIN	= (1u << 7u),
-	ADM_CMD_UNPIN	= (1u << 8u),
-	ADM_CMD_DELETE	= (1u << 9u),
-	USR_CMD_REPORT	= (1u << 10u),
-	USR_CMD_DELVOTE	= (1u << 11u),
+	ADM_CMD_BAN	= (1u << 1u),
```
Comment (komori-k) [inline src/gwbot/modules/003_admin/admin.c:39]: Shouldn't we start from `(1u << 0u)`?

**Review by komori-k — CHANGES_REQUESTED** [review]
Everything looks good to me, except for the enumeration start number.

File: src/gwbot/modules/003_admin/admin.c (hunk 1 above)
Comment (ammarfaizi2) [reply src/gwbot/modules/003_admin/admin.c:39]: Oh right, my bad, will push a revision for this.

**Review by komori-k — APPROVED** [review]
LGTM

**Pull request merged**